src/guides/bdscript2.md

```diff
@@ -1,15 +1,13 @@
 # BDScript 2
-The new edition of BDScript.
 
 ## Introduction
-As you probably know, the default scripting language in the app is **BDScript**, there is also an alternative edition called **BDScript Unstable**.\
-**BDScript 2** has been created with intention of enhancing its capabilities and fixing some of the problems the other versions had.\
-The default edition of **BDScript** has one big issue, commands like `$sum[$sum[3;2];1]` don't work. The reason it doesn't work is because **BDScript** has a predefined order for executing functions.
-I won't go into the nitty-gritty details how exactly it works so, just so you know there are certain function combinations that don't work together.\
-In order to fix the issue, we've created a new BDScript edition called **BDScript Unstable**. It executes function in a command from bottom to top, from right to left.
-It fixed the issue but the new edtition had its own quirks which for some commands could be problematic. That's were **BDScript 2** comes in. This edition executes commands from top to bottom, from left to right *(basically just the way you read most of the books)*.
-Besides that, **BDScript 2** has new features like `$eval[]`, `$try` and `$catch` and more.\
-This edition is still rough on edges but we intend to polish it and make it the default choice in the app.
+**BDScript 2** is the default scripting language in-app _(as of October 2021)_. It has been created with intention of enhancing it's capabilities and fixing some of the problems the other versions had.
```

Comment:
Move `in-app` in front of `scripting` to create `default in-app scripting language`
Also, should be `the problems previous versions had.`

```suggestion
**BDScript 2** is the default in-app scripting language _(as of October 2021)_. It has been created with intention of enhancing its capabilities and fixing some of the problems previous versions had.
```